runtime/gc_realtime/RealtimeGC.hpp

```diff
@@ -166,37 +170,38 @@ class MM_RealtimeGC : public MM_GlobalCollector
 		return (_extensions->nonDeterministicSweep && !isCollectorSweepingArraylets());
 	}
 
-	virtual uintptr_t getVMStateID() { return OMRVMSTATE_GC_COLLECTOR_METRONOME; };
+	uintptr_t getVMStateID() { return OMRVMSTATE_GC_COLLECTOR_METRONOME; };
```

Comment:
I believe this method should stay `virtual`. It overrides parent collector method.